packages/frontend/src/ScreenController.tsx

```diff
@@ -138,13 +138,15 @@ export default class ScreenController extends Component {
     if (!dontStartIo) {
       await BackendRemote.rpc.startIo(accountId)
     }
-    runtime.setDesktopSetting('lastAccount', accountId)
-    BackendRemote.rpc.getSystemInfo().then(info => {
-      log.info('system_info', info)
-    })
-    BackendRemote.rpc.getInfo(accountId).then(info => {
-      log.info('account_info', info)
-    })
+    await Promise.all([
+      runtime.setDesktopSetting('lastAccount', accountId),
+      BackendRemote.rpc.getSystemInfo().then(info => {
+        log.info('system_info', info)
+      }),
+      BackendRemote.rpc.getInfo(accountId).then(info => {
+        log.info('account_info', info)
+      }),
+    ])
```

Comment:
```suggestion
    await runtime.setDesktopSetting('lastAccount', accountId)
    BackendRemote.rpc.getSystemInfo().then(info => {
      log.info('system_info', info)
    })
    BackendRemote.rpc.getInfo(accountId).then(info => {
      log.info('account_info', info)
    })
```

The latter two are purely for debugging, there is no need to `await` them.

Reply:
It's weird, but if I take any of these out of Promise.all it's easier for me to reproduce settingsStore()[0] being null which is the reason for app to crash. It might be actually something else, and `await`-ing these 2 buys time for settingsStore to set up. So maybe we also need to have an increased setTimeout for Settings the same way I did for "All Media".

To be fair with my changes I got a couple of crashes too if I go crazy

Reply:
Looks like I've found a solution, gonna push soon